src/test/resources/build-parent/pom-common.xml

```diff
@@ -9,6 +9,10 @@
     <version>1.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
+    <properties>
+        <version.jgit>${version.jgit}</version.jgit>

```

Comment:
Isn't this redundant? If the property is already set, there is no need to set it again, to its own value!?

Reply:
I guess I should add a comment. The version is needed in `quarkus-scenario/bom/pom.xml` in `template.zip` (see also diff in PR description).
It must be the same version as the one by the GIB project itself because the integration tests work with a small dedicated local maven repo that is derived from the main GIB `pom.xml`.